src/material/core/focus_indicator/_focus-indicator.scss

```diff
@@ -0,0 +1,61 @@
+@import '../theming/theming';
+
+@mixin mat-strong-focus-indicators() {
+  // Base styles for focus indicators.
+  .mat-focus-indicator::before {
+    border-radius: 4px;
+    border-width: 2px;
+    border-style: solid;
+    box-sizing: border-box;
+    position: absolute;
+    pointer-events: none;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+  }
+
+  // Default insets/outsets for focus indicators.
+  .mat-focus-indicator {
+    &.mat-button-base::before,
+    &.mat-card::before,
+    &.mat-sort-header-button::before {
+      margin: -4px;
```

Comment:
Maybe the `-4px` should be based on a variable (e.g. border width * -2px).

Reply:
Are you thinking I'd pull various variables from button, card, sort-header, tab, etc into this file that I'd then use to adjust the focus indicator inset/offset accordingly? Or are you thinking that these margins should be based on the width of the focus indicator border? Just making sure I understand what you're asking.

Reply:
I mean just having a private variable inside the mixin so that everything updates if we change the width of the border.